packages/core/EventManager.ts

```diff
@@ -673,7 +686,7 @@ export default class EventManager {
       await Promise.all(
```

Comment:
Shouldn't se just delete this code otherwise calendar events could be created twice.

One due to this and the other due to createAllCRMEvents(as that includes both old and new CRM credentials)